drv/local-vpd/src/lib.rs

```diff
@@ -163,4 +184,21 @@ pub fn read_config_into(
     Err(err(LocalVpdError::NoRootChunk))
 }
 
+/// Read the Oxide barcode tag and parse it.
+///
+/// Supports `0XV1` and `0XV2` formats.
+pub fn read_oxide_barcode(
```

Comment:
Now that we've got `drv-packrat-vpd-loader`, I wonder if this function should be moved into there.

Moving the function would remove the dependency on `oxide_barcode` and keep `drv-local-vpd` as a thin raw-byte-reader.